text/chapter8.md

```diff
@@ -6,58 +6,11 @@ In the last chapter, we introduced applicative functors, an abstraction which we
 
 The goal of this chapter is to explain why monads are a useful abstraction, and their connection with _do notation_.
 
-## Expose Only Unit Tests for Exercise You Are Implementing
```

Comment:
Chapter 2 has this covered, and it's not repeated for any of the other chapters, so I don't think it's required in Ch8